### DISCUSSION TITLE
Best Univariate Forecaster with Exogeneous data

### DISCUSSION THREAD

**Discussion by arunkumar790** [alan-turing-institute/sktime#6683] [Q&A] [answered]
Hi,

I recently started to use sktime. I am interested in exploring the best Univariate Forecaster that can handle Exogeneous data. I request your suggestion, it would be a great help. Thanks

**Comment by fkiraly** [comment, marked as answer]
There's no such thing as a "universally best" forecaster - it will depend on your data regime generally.

For instance, less complex estimators work better on smaller datasets, and more complex estimators (deep learning, tuning, automl) may overfit on smaller data sets.

It is also relevant whether you have other instances of time series that you could train on.

General best practice suggestions are (after cleaning and exploration of your data) to start with simple models, establish baseline performances, and state clearly how you are planning to measure performance and compare different estimators.

In fact, reproducible working practice and evaluation setup are quite crucial if you are trying to systematically explore the space of models.

Do you have more details on your use case?

**Reply by arunkumar790** [reply]
@fkiraly Thanks for your quick reply. I will start with simple models and then explore a bit more complex ones. The use case is for the manufacturing industry univariate time series prediction, having exogenous will be an added advantage.
